src/main/java/baron/Baron.java

```diff
@@ -0,0 +1,41 @@
+package baron;
+
+import task.TaskList;
+import parser.Parser;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Baron {
+    public static void main(String[] args) throws IllegalArgumentException {
+        greetUser();
+        TaskList tasks = new TaskList();
+        FileStorage.loadFile();
+        try {
+            getUserInput(tasks);
+        } catch (IllegalArgumentException e) {
+            System.out.println("Command cannot be empty. Please enter a valid command.");
+        } catch (IOException e) {
+            System.out.println("Error: " + e.getMessage() + "\n");
+        }
+    }
```

Comment:
It is advisable to have logically linked code to be put together, and other part separated by a line break.


```suggestion
    public static void main(String[] args) throws IllegalArgumentException {
        greetUser();
        
        TaskList tasks = new TaskList();
        
        FileStorage.loadFile();
        
        try {
            getUserInput(tasks);
        } catch (IllegalArgumentException e) {
            System.out.println("Command cannot be empty. Please enter a valid command.");
        } catch (IOException e) {
            System.out.println("Error: " + e.getMessage() + "\n");
        }
    }
```


Reference: https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically